### PR TITLE
Update pt-BR language

### PR DESCRIPTION
**Pull request by dannyloxavier**
Reviewed and updated

### CODE REVIEW

File: i18n/pt-BR.toml [hunk 1]
```diff
@@ -80,17 +102,14 @@ other = "Nenhum resultado encontrado"
 [lunrLanguageCode]
 other = "pt"
 
-[lunrLanguageLib]
-other = "lib/lunr/lunr.pt.js"
-
```
Comment (HEIGE-PCloud) [inline i18n/pt-BR.toml]: Why removing the lunr library?

Comment (dannyloxavier) [reply i18n/pt-BR.toml]: I did not saw it in 'en' language. If it should stay, I'll undo this.

Comment (dannyloxavier) [reply i18n/pt-BR.toml]: Done!

**Comment by dannyloxavier** [comment]
I am new at git, so I don't know if I need to make a new pull request, after the commit solving my mistake.

**Comment by HEIGE-PCloud** [comment]
> I am new at git, so I don't know if I need to make a new pull request, after the commit solving my mistake.

No, you don't, you have pushed your new commits to this branch successfully.

**Review by HEIGE-PCloud — APPROVED** [review]
It looks good to me, thanks for your contribution! :)

File: i18n/pt-BR.toml (hunk 1 above)
Comment (HEIGE-PCloud) [reply i18n/pt-BR.toml]: The [lunr search](https://lunrjs.com/) requires extra plugins to [support languages other than English](https://lunrjs.com/guides/language_support.html), so an extra field is needed.

**Pull request merged**